README.md

```diff
@@ -13,53 +13,221 @@ The toolkit grew out of a previous project called [Warcbase](https://github.com/
+
+To run Apache Spark Shell:
+
+```
+spark-2.4.4-bin-hadoop2.7/bin/spark-shell
+```
+
+You should see something like:
+
+```
+Spark context Web UI available at http://10.0.1.44:4041
+Spark context available as 'sc' (master = local[10], app id = local-1572059365536).
+Spark session available as 'spark'.
+Welcome to
+      ____              __
+     / __/__  ___ _____/ /__
+    _\ \/ _ \/ _ `/ __/  '_/
+   /___/ .__/\_,_/_/ /_/\_\   version 2.4.4
+      /_/
+         
+Using Scala version 2.11.12 (OpenJDK 64-Bit Server VM, Java 1.8.0_222)
+Type in expressions to have them evaluated.
+Type :help for more information.
+
+scala> 
+```
+
 ## Getting Started
 
-### Easy
+### Archives Unleashed Toolkit with Spark Shell
 
-If you have Apache Spark ready to go, it's as easy as:
+There are a two options for loading the Archives Unleashed Toolkit. The advantages and disadvantages of using either option are going to depend on your setup (single machine vs cluster):
 
+```shell
+$ spark-shell --help
+
+  --jars JARS                 Comma-separated list of jars to include on the driver
+                              and executor classpaths.
+  --packages                  Comma-separated list of maven coordinates of jars to include
+                              on the driver and executor classpaths. Will search the local
+                              maven repo, then maven central and any additional remote
+                              repositories given by --repositories. The format for the
+                              coordinates should be groupId:artifactId:version.
 ```
-$ spark-shell --packages "io.archivesunleashed:aut:0.18.0"
+
+#### As a package
+
+Release version:
+
+```shell
+$ spark-shell --packages "io.archivesunleashed:aut:0.18.1"
 ```
 
-### A little less easy
+HEAD (built locally):
+
+```shell
+$ spark-shell --packages "io.archivesunleashed:aut:0.18.2-SNAPSHOT"
+```
 
-You can download the [latest release here](https://github.com/archivesunleashed/aut/releases) and include it like so:
+#### With an UberJar
 
+Release version:
+
+```shell
+$ spark-shell --jars /path/to/aut-0.18.1-fatjar.jar
+```
+
+HEAD (built locally):
+
+```shell
+$ spark-shell --jars /path/to/aut-0.18.2-SNAPSHOT-fatjar.jar
+```
+
+### Archives Unleashed Toolkit with PySpark
+
+To run PySpark with the Archives Unleashed Toolkit loaded, you will need to provide PySpark with the Java/Scala package, and the Python bindings. The Java/Scala packages can be provided with `--packages` or `--jars` as described above. The Python bindings can be [downloaded](https://github.com/archivesunleashed/aut/releases/download/aut-0.18.1/aut-0.18.1.zip), or [built locally](#Introduction) (the zip file will be found in the `target` directory.
+
+#### As a package
+
+Release version:
+
+```shell
+$ export PYSPARK_PYTHON=/path/to/python; export PYSPARK_DRIVER_PYTHON=/path/to/python; /path/to/spark/bin/pyspark --py-files aut-0.18.1.zip --packages "io.archivesunleashed:aut:0.18.1"
+```
+
```

Comment:
I think this command really throws people - @SamFritz might have had trouble with this too (not sure if she ever got it working). Maybe an example command? Or suggesting they run `which python` to get their Python path? 

(putting a lot of this in a README might be a bridge too far)

Reply:
i.e. here's my command

```
export PYSPARK_PYTHON=/Users/ianmilligan1/anaconda/bin/Python; export PYSPARK_DRIVER_PYTHON=/Users/ianmilligan1/anaconda/bin/Python; ./bin/pyspark --py-files /Users/ianmilligan1/dropbox/git/aut/target/aut.zip --packages "io.archivesunleashed:aut:0.18.1"
```

(the important thing is I think the `which python`, as finding a python path can really throw users for a loop)